apps/api/src/app/controllers/command/command-payload-schemas.spec.ts

```diff
@@ -13,8 +12,6 @@ describe('command payload schemas', () => {
 
         await testModule.init();
 
-        await testModule.get(DynamicDataTypeFinderService).bootstrapDynamicTypes();
```

Comment:
It's good to see these gone. We should make sure this is called in the app module once and at the appropriate time.